filament/backend/src/metal/MetalEnums.h

```diff
@@ -435,6 +435,21 @@ inline MTLTextureSwizzleChannels getSwizzleChannels(TextureSwizzle r, TextureSwi
             getSwizzle(a));
 }
 
+inline bool isStencilFormat(MTLPixelFormat format) {
+#if TARGET_OS_OSX
+    if (@available(macOS 10.11, macCatalyst 13.0, *)) {
```

Comment:
`TARGET_OS_OSX` is `false` on Mac Catalyst, the condition to use is `(TARGET_OS_OSX || TARGET_OS_MACCATALYST)`. 
Also, I don't think we need the `@available()` runtime checks, minimum target versions of the lib already satisfy the conditions.